android/app/google-services.json

```diff
@@ -8,20 +8,12 @@
   "client": [
     {
       "client_info": {
-        "mobilesdk_app_id": "1:366914936426:android:7d2393ccb8b3bdda1a3413",
+        "mobilesdk_app_id": "1:366914936426:android:978896cb9d8996661a3413",
         "android_client_info": {
-          "package_name": "com.joinseeds.seedswallet"
+          "package_name": "com.joinseeds.parq"
```

Comment:
i dont think this change is supposed to be here

Reply:
intresting this chenages was made by the FlutterFire ClI tool I only pick the seeds project in available options to confi the Crashlytics.
Should I keep that name @n13 ?

Reply:
Why do we have 2 android projects?
![image](https://user-images.githubusercontent.com/42857405/175855723-f3d8a244-416a-48a1-9970-b5d084897c75.png)

Reply:
Please don't change that. 

This is correct

package_name": "com.joinseeds.seedswallet

Why we have multiple firebase projects - Igor did this, we can delete them.

Reply:
@RaulUrtecho is there's strange merge errors, I'd tend to just start over.